### DISCUSSION TITLE
Using Multiple queues in configuration

### DISCUSSION THREAD

**Discussion by aleksandarTenevv** [stackkit/laravel-google-cloud-tasks-queue#99] [Q&A]
Hello,

I am trying to set this package to use multiple queues but I haven't found anything on how to achieve that or is it possible to be set.
The only solutions that came to my mind was to create another connection in queue.php config and dispatch the jobs on that connection. This solution is creating the jobs in a different Cloud Tasks queue but when the /handle-task endpoint is called I get 404 response. The default queue is executing tasks on the same endpoint and that works normal. 
Is there any way to achieve this using one connection config with multiple queues?

**Comment by marickvantuil** [comment]
Hi,

Perhaps I am misunderstanding but it's possible to use different queues:

```php
ProcessPodcast::dispatch()->onQueue('emails');

// ...or...

class ProcessPodcast implements ShouldQueue
{
    use Dispatchable, InteractsWithQueue, Queueable, SerializesModels;
 
    /**
     * Create a new job instance.
     */
    public function __construct()
    {
        $this->onQueue('processing');
    }
}
```

Or is this not what you are trying to achieve?

**Comment by Airgerr** [comment]
It is not clear for somebody reading the MD that the queue that is being set is just the "default" queue. That is probably where the question is coming from. ;)
